raspbian/applications/vcgencmd.md

```diff
@@ -87,6 +87,8 @@ Displays the content of the One Time Programmable (OTP) memory, which is part of
 
 Reports on the amount of memory allocated to the ARM cores `vcgencmd get_mem arm` and the VC4 `vcgencmd get_mem gpu`.
 
+**Note:** On a Raspberry Pi 4 with greater than 1GB of RAM, the `arm` option is inaccurate. This is because the GPU firmware which implements this command is only aware of the first GB of RAM on the system, so the `arm` setting will always return 1GB minus the `gpu` memory value. To get an accurate report of the amount of ARM memory, use one of the standard Linux commands, such as `free` or `cat /proc/meminfo`
```

Comment:
"the first GB of RAM"

IMHO seeing `GB` like that without a number before it looks very odd - perhaps it should be expanded to "the first gigabyte of RAM" ?